### ISSUE TITLE
Change the default of Right-mouse Scrolling

### ISSUE THREAD

**Issue by Arular101** [OpenRA/OpenRA#14551]
Currently, the default for Right-mouse Scrolling is: Disabled. I propose to change the default to: Joystick. This is iirc for every C&C game from Tiberian Sun on.

**Comment by penev92** [comment]
IIRC middle-mouse scrolling is what's modern now, and right-mouse scrolling can conflict with right-click orders if the player isn't precise enough with the clicking. As a result the defaults are set up so that there are no issues.

**Comment by dragunoff** [comment]
I agree with @penev92  here. Right-mouse scrolling works better with left-click orders (what the original CnC games had) and players can change both settings if they want to replicate the original behavior.

**Comment by Arular101** [comment]
> IIRC middle-mouse scrolling is what's modern now

The middle-mouse button is used to rotate the camera in 3D C&C’s (tested with Red Alert 3). 2D C&C’s didn’t had a function for the middle-mouse button (tested with Tiberian Sun).

> right-mouse scrolling can conflict with right-click orders if the player isn't precise enough with the clicking.

If by any chance the player makes a mistake, the only consequence of such action is that the player gives a move order where he didn’t want it to be. Luckily, this can easily be corrected by giving another order to the right spot. A bigger consequence is there if a mistake is made with the left-mouse button. Because, when the player carefully selected a squad of units, a miss-click will deselect everything. It takes more time to correct such mistake.

We should also consider that not a lot of people knows about this feature in OpenRA. I only know about this because I lurked around at GitHub when this feature was added (and that it is called “joystick”). I don't have any statistics, but I can say that in a lot of OpenRA YouTube videos and streams people scroll around on the edges or with the keyboard. But when this function is present in the game, people will use the right-mouse scrolling (that includes Tiberian Sun and Red Alert 2 with the left-mouse orders and Tiberium Wars and Red Alert 3 with the right-mouse orders).

The UI in OpenRA is modernized to how it looks like in C&C3 and RA3. This change is a similar thing and should really be considered.

And if by any chance, someone really dislikes this, then the player can search the settings to disable it. But if a player doesn’t know this feature exist, then he will not search for it.

> Right-mouse scrolling works better with left-click orders (what the original CnC games had)

The right-click orders goes actually better with the right-mouse scrolling! The player can’t deselect with the right-mouse button like said above. So no mistakes can be made compared to left-click orders, which can deselect all the units.

I hope I can convince you guys ^^

**Comment by Kanar** [comment]
I agree with penev92 as well.

**Comment by matjaeck** [comment]
IMO the discussion points out that the default setting for right click scrolling is good and I don't think that this issue will go anywhere. Changing the default will affect many players negatively, leaving it as it is does not affect the players who want to change this setting negatively.
